machina/templates/machina/forum/forum_list.html

```diff
@@ -72,16 +72,7 @@ <h3 class="m-0 card-title h5 text-dark">{% trans "Forums" %}</h3>
       <div class="py-2 col-md-1 d-none d-md-block text-center forum-count">{{ node.posts_count }}</div>
       <div class="py-2 col-md-2 col-sm-3 d-none d-sm-block forum-last-post">
         {% if node.last_post %}
-        {% if node.last_post.poster %}
-        {% url 'forum_member:profile' node.last_post.poster_id as poster_url %}
-        {% blocktrans trimmed with poster_url=poster_url username=node.last_post.poster|forum_member_display_name %}
-        By: <a href="{{ poster_url }}">{{ username }}</a>
-        {% endblocktrans %}
-        {% else %}
-        {% blocktrans trimmed with poster_username=node.last_post.username %}
-        By: {{ poster_username }}
-        {% endblocktrans %}
-        {% endif %}
+        {% include "partials/posted_by.html" with poster=node.last_post.poster username=poster_username %}
```

Comment:
`{{ poster_username }}` doesn't exist in the context. Shouldn't it be `{{ node.last_post.username }}` instead? Otherwise a blank string will be displayed for anonymous users. This question also applies to all the where the `posted_by.html` partial was inserted. 😉

Reply:
Thank you for you feedback, and sorry about that, it is now fixed !